scripts/tools/tools-schema.json

```diff
@@ -50,7 +50,7 @@
             "properties": {
                 "language": {
                     "type": "string",
-                    "description": "Provide information on how other can integrate with the tool in case it is possible. If your code generator generates Python code, then specify Python, even if code generator itself is written in JavaScript. If we forgot about some language then please add it through a pull request to AsyncAPI website repository.",
+                    "description": "Refers to the language that users can use in the 'runtime'. That is, for generators it should be the language that the generator generates, not the language the generator is written in, itself. But, for example, for Bundler library, even though it operates on YAML and JSON files, more important for users is the fact that the library is written in TypeScript as this information tells them if they can integrate this library into their 'runtime'. In other words, if users have TypeScript codebase and they would like to have a bundling mechanism, they need to know if they can integrate Bundler library into their TypeScript codebase. This property provides information if a user can use given library/script/API specifically in his code. If we forgot about some language then please add it through a pull request to AsyncAPI's 'website' repository.",
```

Comment:
Can this be made quite shorter or in brief? because we have to show this description in UI also.

Reply:
It was shorter already and it was absolutely un-understandable to a person who sees this property for the first time in his life.
@alequetzalli do you have ideas how to rephrase it? Coining this formulation, so it keeps technical correctness and the clearest explanation possible, took a week of tight communication with @derberg, but maybe something is too repetitive in it?

Reply:
"to a person who sees this property for the first time in **THEIR** life."  😜